src/Codegen/Types.hack

```diff
@@ -176,23 +175,26 @@ enum IO: string as string {
  * Get the type aias for the given type structure.
  */
 function type_structure_to_type_alias<T>(TypeStructure<T> $ts): string {
+    $prefix = ($ts['nullable'] ?? false) ? '?' : '';
```

Comment:
We need this to handle vecs of nullables

Reply:
Just make sure we're still doing the right thing for regular (output) object types (I think this code is shared by both) -- but I don't see any changes to generated output objects so it's probably good.

Reply:
This was double wrapping in these cases: https://github.com/mwildehahn/hack-graphql/pull/97/commits/ebfdea44f9cd468b86322c8b0856128c56f02215 but `input_type` calls `unwrap_type` recursively so it handled it.